util/HalideTraceViz.cpp

```diff
@@ -621,8 +621,8 @@ int run(int argc, char **argv) {
                     }
 
                     // Stores are orange, loads are blue.
-                    uint32_t color = p.event == 0 ? 0xffffdd44 : 0xff44ddff;
-
+                    const auto color = p.event == 0 ? 0xff44ddff : 0xffffdd44;
```

Comment:
In little-endian rgba, 0xffffdd44 is blue, and 0xff44ddff is orange. You seem to be changing it to little-endian bgra. I'm not sure if the rest of the code will work in that case. E.g. the thing that shows color images will probably show them with incorrect colors, because it puts them out in rgba order.